scripts/rollup/build.js

```diff
@@ -363,6 +374,7 @@ function getPlugins(
     isUMDBundle && entry === 'react-art' && commonjs(),
     // Apply dead code elimination and/or minification.
     isProduction &&
+      !skipClosureTranspilation &&
```

Comment:
what's the reason for skipping closure for bun? beyond just minifying there is inlining and code elimination uses for closure. For instance in the prod bun build the `enableFloat` flag is retained in a few places when they would be elided with the closure step.

Reply:
I've changed this to use closure for the prod build. I was running into issues with resolving bare module specifiers 
 `import ... from "react"`) when the rollup output format is `es`. This isn't a problem when targeting `cjs` since  `process_common_js_modules` is `false`. Switching to a `cjs` target solved the problem, and Bun can handle both formats anyway...ESM was just a nice-to-have.

Reply:
Ah, interesting. are there any material benefits to the esm option or just that it is "the future" and would be nice to use it b/c Bun supports it?

Reply:
Mostly that it's "the future" :) But Bun transpiles every file anyway so perf is about the same regardless.